### ISSUE TITLE
Unnecessary .DS_Store files

### ISSUE THREAD

**Issue by zhulduz** [raphamorim/react-tv#46]
Hi,
example app has files `.DS_Store`.
After creating with command `react-tv init <app-name>` .

**Comment by raphamorim** [comment]
hi @zhulduz, thanks for reporting it.
`react-tv init` has been rewritten for the next react-tv version. You should not have that problem any more.

You can track the release status on this PR: https://github.com/raphamorim/react-tv/pull/42

**Comment by raphamorim** [comment]
I'll close this issue after merge #42 PR :octocat:

**Comment by raphamorim** [comment]
I published `react-tv@0.3.0-alpha.2` today:
The `react-tv init` has changed:

```bash
$ yarn global add react-tv@0.3.0-alpha.2 
$ react-tv init my-project && cd my-project
$ yarn && yarn start-dev
```

Should solve it, if not just comment here and I'll reopen this issue :octocat: :)
